cmd/config/list_keys.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"github.com/pingidentity/pingcli/cmd/common"
+	config_internal "github.com/pingidentity/pingcli/internal/commands/config"
+	"github.com/pingidentity/pingcli/internal/configuration/options"
+	"github.com/pingidentity/pingcli/internal/logger"
+	"github.com/spf13/cobra"
+)
+
+const (
+	listKeysCommandExamples = `  pingcli config list-keys
```

Comment:
I think the rest of the config command examples have descriptions, so those would be good to add here.